### PR TITLE
Ross change

### PR DESCRIPTION
**Pull request by tjstebbing**
Updated Ross' position in about files for all translations, posted announcement about Ross moving to advisor

### CODE REVIEW

File: content/announcements/2022-02-24-update-ross.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title = "Ross Nicoll Stepping into an Advisor Role for the Dogecoin Foundation"
+date = "2022-02-24"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+
+Recently Ross [made public his decision](https://rnicoll.name/posts/2022-02-16-stepping-away/) to 
+step back from involvement in the Foundation as a Director, and more generally from 
+active development in the crypto-space.
```
Comment (exu3) [inline content/announcements/2022-02-24-update-ross.md:11]:
```suggestion
active development in the crypto space. 
```

File: content/announcements/2022-02-24-update-ross.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Recently Ross [made public his decision](https://rnicoll.name/posts/2022-02-16-stepping-away/) to 
+step back from involvement in the Foundation as a Director, and more generally from 
+active development in the crypto-space. 
+
+While this is news to many in the community, Ross had made his intentions clear to 
+the Core developers almost a year ago and has been working quietly behind the scenes 
+to create a sustainable transition for his role in the Dogecoin project. 
+
+With this in mind, Ross has been instrumental in the re-establishment of the Dogecoin 
+Foundation in 2021. The Foundation has been set-up to ensure that Dogecoin prospers
```
Comment (exu3) [inline content/announcements/2022-02-24-update-ross.md:18]:
```suggestion
Foundation in 2021. The Foundation has been set up to ensure that Dogecoin prospers 
```

File: content/announcements/2022-02-24-update-ross.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
++++
+title = "Ross Nicoll Stepping into an Advisor Role for the Dogecoin Foundation"
+date = "2022-02-24"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+
+Recently Ross [made public his decision](https://rnicoll.name/posts/2022-02-16-stepping-away/) to 
+step back from involvement in the Foundation as a Director, and more generally from
```
Comment (exu3) [inline content/announcements/2022-02-24-update-ross.md:10]:
```suggestion
step back from involvement in the Foundation as a Director and more generally from 
```

**Review by exu3 — COMMENTED** [review]
Gave it a spin & the CSS changes look good! Just a few grammar suggestions:

**Review by exu3 — DISMISSED** [review]

File: content/announcements/2022-02-24-update-ross.md (hunk 1 above)
Comment (exu3) [reply content/announcements/2022-02-24-update-ross.md:11]: (https://github.com/dogecoinfoundation/foundation.dogecoin.com/blob/377b0a4ed426e3b5e9bd17c9c56d211525d8a298/content/advisories/2021-08-31-advisory-dogecoin2.md#L12) is written as 2 words.

**Comment by bobdos** [comment]
@tjstebbing 

* i updated zh-cn and zh-tw for about files
* other translation files all look good

**Pull request merged**